rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :name
+
+  def initialize(date, book, name = 'unknown')
+    @date = date
+    @book = book
+    @name = name
```

Comment:
- [ ] @fatmahussein Kindly consider ensuring rental is adding itself to the @book  and @name in order to achieve the 'belongs-to' association. I would also recommend changing the variable @name to @person(this is because @name is supposed to represent the person and will make the code easier to understand even in the future).